### DISCUSSION TITLE
I have two discussions marked as answers but still no galaxy brain badge, why is that?

### DISCUSSION THREAD

**Discussion by DaviZCodes** [community/community#131459] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have two questions marked as answers and still no badge.
1. https://github.com/orgs/community/discussions/131024
2. https://github.com/orgs/community/discussions/102934

I searched online and it says I cannot mark my own question as an answer, but I have never done that. 

I should have the badge on my profile, no?

**Comment by davevad93** [comment, marked as answer]
Hi @DaviZCodes ,

Unfortunately the `Galaxy Brain` badge is no longer earnable via Community Discussions, for more information see [here](https://github.com/orgs/community/discussions/106536). However you can still get the achievement by answering open discussions in other GitHub's repos.
